ynr/apps/elections/uk/templates/candidates/person-view.html

```diff
@@ -198,6 +198,8 @@ <h2>Photo Credit:</h2>
   <p class="person__photo-credit">
     {% include 'candidates/_photo-credit.html' with image=person.person_image_model %}
   </p>
+  <i>I am {{ person.name }}, and I would like to remove this photo:</i>
+    <a href="mailto:hello@democracyclub.org.uk" class="button">Request photo removal</a>
```

Comment:
We could have this trigger our backend to send an email for the user rather than use a `mailto` if we wanted to be a bit more clever

Reply:
Much better idea 👍

Reply:
Would only want authenticated users or anyone to have this ability?